src/main/java/de/rubenmaurer/punk/util/Terminal.java

```diff
@@ -171,6 +171,14 @@ public static void printLastResponse(String... responses) {
                 .single("count", String.valueOf(responses.length)).render());
     }
 
+    public static void printLog(String message) {
+        if (message != null && !message.isEmpty()) {
+            System.err.println(Template.get("LOG")
+                .single("type", new SimpleDateFormat("yyyy-MM-dd hh:mm:ss").format(new Date()))
+                .single("message", message).render());

```

Comment:
I overlooked the `log()` method inside the `Template` class, this would make the call easier.